sdk/translation/azure-ai-translation-text/tsp-location.yaml

```diff
@@ -1,3 +1,3 @@
 directory: specification/translation/Azure.AI.TextTranslation
-commit: 9b1d6b68cb4ab26724d6e813b7649a40902d622d
-repo: Azure/azure-rest-api-specs
+commit: fc277f3e8b299b84a8a39105353cdb5a5df4b890
+repo: MikeyMCZ/azure-rest-api-specs
```

Comment:
For release, we should use the commit id from the main repo.

Reply:
Will do - I am waiting for reviews in other languages as well to make sure to have same version everywhere (if there will be any changes needed in TypeSpec repo).